cfpb/consumerfinance.gov
# Remove large pull quote

I noticed the large pull quote is only used on https://www.consumerfinance.gov/data-research/consumer-complaints/

This pattern is very old and goes back to 2014 (added in https://github.com/cfpb/cf-typography/pull/17)

We have the "is large" option in Wagtail to support the one use case in the CCDB landing page.

Consulting with @natalia-fitzgerald, it was decided it would be fine for this page to have the regular pull quote, which I have updated on the live site.

## Removals

- Large pull quote.


## How to test this PR

1. Run `python cfgov/manage.py migrate` and edit a page in Wagtail that has a quote, such as the CCDB landing page, and see there is no longer an "is large" option for the pull quote. 


## Screenshots

Before:

<img width="1211" alt="Screenshot 2023-10-25 at 1 03 55 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/6c0e33be-3d3c-4435-a7e2-2b19c76ef2c8">


After:

<img width="1217" alt="Screenshot 2023-10-25 at 1 03 42 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/862fd2cd-1578-411f-ad0a-7d9ad72c1c42">
